Src/FluentAssertions/Formatting/DefaultValueFormatter.cs

```diff
@@ -28,22 +25,29 @@ public virtual bool CanHandle(object value)
             return true;
         }
 
-        /// <inheritdoc />
-        public string Format(object value, FormattingContext context, FormatChild formatChild)
+        public void Format(object value, FormattedObjectGraph formattedGraph, FormattingContext context, FormatChild formatChild)
         {
             if (value.GetType() == typeof(object))
             {
-                return $"System.Object (HashCode={value.GetHashCode()})";
+                formattedGraph.AddFragment($"System.Object (HashCode={value.GetHashCode()})");
+                return;
             }
 
-            string prefix = context.Options.UseLineBreaks ? Environment.NewLine : string.Empty;
```

Comment:
I can't really tell if it makes a difference and if so, if it's better or worse.
`UseLineBreaks` is ignored in `WriteTypeAndMemberValues`.

Reply:
If the formatter needs to render the object with its properties, it will always do that as a multi-line representation.